packages/vue3-vite/src/generators/vue3-vite/files/jest.config.ts.template

```diff
@@ -1,19 +1,19 @@
 module.exports = {
   displayName: '<%= projectName %>',
-  preset: '<%= offsetFromRoot %>jest.preset.js',
   globals: {
     'vue-jest': {
       tsConfig: './<%= projectRoot %>/tsconfig.spec.json',
     },
   },
   transform: {
     '^.+\\.vue$': '@vue/vue3-jest',
-    '^.+\\.[jt]sx?$': 'ts-jest',
+    "^.+\\.jsx$": 'babel-jest',
+    '^.+\\.tsx?$': 'ts-jest',
   },
-  testEnvironment: 'jsdom',
-  moduleNameMapper: {
-    '^@/(.*)$': '<rootDir>/src/$1',
+  testEnvironmentOptions: {
```

Comment:
I had to add this to get everything to work, I saw the fix [here](https://github.com/vuejs/vue-jest/issues/479#issuecomment-1163421581)